apertium-tam.tam.rlx

```diff
@@ -1,15 +1,71 @@
-# Firstly, we need to define what tags should be considered sentence delimiters. For this example, only full stop is set as delimiter.
-DELIMITERS = "<.>" ;
+DELIMITERS = "<.>" "<!>" "<?>" "<...>" ;
 
-# We can define sets for common tag clusters
-LIST Case = (n s) (n p) ; # A set that matches either a Noun Singular or a Noun Plural
+SETS
+
+LIST N = n;
+LIST Adj = adj;
+LIST Adv = adv;
+LIST Verb = vblex;
+LIST Vbser = vbser;
+LIST Prn = prn;
+LIST MaleNoun = (n m);
+LIST FemaleNoun = (n f);
+LIST NorPrn = (n) or (prn);
```

Comment:
I think what you want is `LIST NorPrn = n prn;`

Reply:
Yup.